TechnicoServer/TechnicoDomain/Services/RepairService.cs

```diff
@@ -0,0 +1,153 @@
+﻿using TechnicoDomain.Dtos;
+using TechnicoDomain.Models;
+using TechnicoDomain.Repositories;
+
+namespace TechnicoDomain.Services;
+
+public class RepairService : IRepairService
+{
+    private readonly IRepository<Repair, long> _repairRepository;
+
+    public RepairService(IRepository<Repair, long> repairRepository)
+    {
+        _repairRepository = repairRepository;
+    }
+
```

Comment:
The code that converts Repair to RepairResponseDto is repeated in several methods. This could be simplified with a helper method I think.